cmd/util/ledger/migrations/storage_v6.go

```diff
@@ -191,8 +117,16 @@ func (m *StorageFormatV6Migration) migrate(payloads []ledger.Payload) ([]ledger.
 
 	m.migratedPayloadPaths = make(map[storagePath]bool, 0)
```

Comment:
What is the purpose of this? I see that we add paths when we migrate individual payloads, but I did not quite understand why we check it and panic in the old interpreter's `WithStorageReadHandler`

Reply:
This is a sanity check to make sure we don't re-add deferred payloads as separate payloads, and vise-versa (i.e: already migrated payloads are not re-added as deferred payloads)

Reply:
Added in 8677eb7

Reply:
Ah, that makes sense 👍